app/src/main/java/com/kyphipraniti/fitnesstasks/fragments/CalendarFragment.java

```diff
@@ -139,6 +146,38 @@ private void setupRecyclerView(View view) {
         RecyclerView mRvTasks = view.findViewById(R.id.rvTasks);
         mRvTasks.setAdapter(mTasksAdapter);
         mRvTasks.setLayoutManager(mLinearLayoutManager);
+        mRvTasks.setItemAnimator(new DefaultItemAnimator());
+
+        ItemTouchHelper itemTouchHelper = new ItemTouchHelper(new ItemTouchHelper.SimpleCallback(ItemTouchHelper.UP |
+            ItemTouchHelper.DOWN, ItemTouchHelper.RIGHT){
+
+            @Override
+            public boolean onMove(RecyclerView recyclerView, RecyclerView.ViewHolder viewHolder, RecyclerView.ViewHolder target) {
+                moveItem(viewHolder.getAdapterPosition(),target.getAdapterPosition());
+                return true;
+            }
+
+            @Override
+            public void onSwiped(RecyclerView.ViewHolder viewHolder, int direction) {
+                completedTaskList(viewHolder.getAdapterPosition());
+            }
+
+        });
+        itemTouchHelper.attachToRecyclerView(mRvTasks);
+    }
+
+    private void completedTaskList(int position) {
```

Comment:
i think this should be `completeTask` yeah? looks like it's being used to mark a task as complete.